skipruntime-ts/helpers/src/rest.ts

```diff
@@ -2,14 +2,15 @@ import type { Json, Entry } from "@skipruntime/api";
 
 export type Entrypoint = {
   host: string;
-  port: number;
+  streaming_port: number;
```

Comment:
Leaving for later is no problem. FWIW I only noticed this because reading the code using this I got confused about why it was receiving the streaming port.